src/Console_TEST.cc

```diff
@@ -505,14 +525,16 @@ TEST_F(Console_TEST, Prefix)
   gzmsg << "message" << std::endl;
   gzdbg << "debug" << std::endl;
 
+  common::Console::Root().RawLogger().flush();
+
   // Get the logged content
   std::string logContent = GetLogContent(logPath);
 
   // Check
-  EXPECT_TRUE(logContent.find("**test** [Err]") != std::string::npos);
-  EXPECT_TRUE(logContent.find("**test** [Wrn]") != std::string::npos);
-  EXPECT_TRUE(logContent.find("**test** [Msg]") != std::string::npos);
-  EXPECT_TRUE(logContent.find("**test** [Dbg]") != std::string::npos);
+  EXPECT_TRUE(logContent.find("**test** error") != std::string::npos);
```

Comment:
Is this a behavior change on the formatting of the logs? Should this be included in the migration guide?